src/drivers/github.js

```diff
@@ -273,15 +273,20 @@ class Github {
       }
 
       await exec(
-        `${resolve(
-          workdir,
-          'config.sh'
-        )} --unattended --token "${await this.runnerToken()}" --url "${
-          this.repo
-        }" --name "${name}" --labels "${labels}" --work "${resolve(
-          workdir,
-          '_work'
-        )}" ${single ? ' --ephemeral' : ''}`
+        resolve(workdir, 'config.sh'),
+        '--unattended',
+        '--token',
+        await this.runnerToken(),
+        '--url',
+        this.repo,
+        '--name',
+        name,
+        '--labels',
+        labels,
+        '--work',
+        resolve(workdir, '_work'),
+        // adds `--ephemeral` to the array only if `single` is set
+        ...(single && ['--ephemeral'])
```

Comment:
I don't think that `['thing', ...(false && ['maybe thing']) ]` works

Reply:
```
node
Welcome to Node.js v16.17.0.
Type ".help" for more information.
> ['thing', ...(false && ['maybe thing']) ]
Uncaught TypeError: (false && ["maybe thing"]) is not iterable
>
```
?

Reply:
Yes, major blunder; I was too lazy, fixed with https://github.com/iterative/cml/pull/1275/commits/b89b383eec5d89be0d3bd8bb7441bb2415973384